thirdparty/runtime_environments.go

```diff
@@ -0,0 +1,60 @@
+package thirdparty
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/evergreen-ci/gimlet"
+	"github.com/pkg/errors"
+)
+
+type RuntimeEnvironmentsClient struct {
+	Client  *http.Client
+	BaseURL string
+	APIKey  string
+}
+
+func NewRuntimeEnvironmentsClient(baseURL string, apiKey string) *RuntimeEnvironmentsClient {
+	c := RuntimeEnvironmentsClient{
+		Client:  &http.Client{},
+		BaseURL: baseURL,
+		APIKey:  apiKey,
+	}
+	return &c
+}
+
+// The function getImageNames returns a list of strings containing the names of all images from the runtime environments API.
```

Comment:
nit: Go doc comments for methods typically start with the function name. For example:
`getImages returns the names of all images from the runtime environment API.` You can see more examples in the Go standard library docs, like the doc comment for [Regexp.Find](https://pkg.go.dev/regexp@go1.22.0#Regexp.Find):
> Find returns a slice holding the text of the leftmost match in b of the regular expression. A return value of nil indicates no match.
> func (re *Regexp) Find(b []byte) []byte`